lib/beamchmark/suite.ex

```diff
@@ -92,11 +93,20 @@ defmodule Beamchmark.Suite do
   end
 
   @spec try_load_base(t()) :: {:ok, t()} | {:error, File.posix()}
-  def try_load_base(%__MODULE__{configuration: config}) do
-    with old_path <- Path.join([config.output_dir, @old_suite_filename]),
+  def try_load_base(%__MODULE__{} = suite) do
+    output_dir = output_dir_for(suite)
+
+    with old_path <- Path.join([output_dir, @old_suite_filename]),
          {:ok, suite} <- File.read(old_path),
          suite <- :erlang.binary_to_term(suite) do
       {:ok, suite}
     end
   end
+
+  defp output_dir_for(%__MODULE__{configuration: config} = suite) do
+    scenario_dir = suite.scenario |> Atom.to_string() |> String.replace(".", "")
+    config_dir = "#{config.delay}_#{config.duration}"
```

Comment:
Can we write in docs what you wrote in PR description i.e. where we save results.